lib/internal/errors.js

```diff
@@ -148,6 +148,7 @@ E('ERR_SOCKET_BAD_TYPE',
 E('ERR_SOCKET_CANNOT_SEND', 'Unable to send data');
 E('ERR_SOCKET_BAD_PORT', 'Port should be > 0 and < 65536');
 E('ERR_SOCKET_DGRAM_NOT_RUNNING', 'Not running');
+E('ERR_INVALID_FD', 'fd must be positive integer');
```

Comment:
These are supposed to be kept alphabetical. (See comment on line 112.) Of course, it doesn't help that the SOCKET ones are all after UNKNOWN and the comment says "Add new errors from here..." so not exactly your fault...